controller/api/permissions.py

```diff
@@ -97,7 +97,19 @@ class HasRegistrationAuth(permissions.BasePermission):
     Checks to see if registration is enabled
     """
     def has_permission(self, request, view):
-        return settings.REGISTRATION_ENABLED
+        """
+        If settings.REGISTRATION_MODE does not exist, such as during a test, return True
```

Comment:
I think that a user could accidentally write out a typo for the registration mode. In that case we should be raising an argument error. We could use "open_season" to accept all requests.